src/components/Iframe.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+import Requests from "./Requests";
+import "../styles/iframe.css";
+
+export default class Iframe extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      game: {}
+    };
+  }
+
+  async componentDidMount() {
+    const id = this.props.match.params.id;
+    this.setState({ game: await Requests.get("/games/" + id) });
+    console.log(this.state.game);
+  }
+
+  render() {
+    return (
+      <div>
+        <h1>Game</h1>
+        <div className="iframe-container">
+          <iframe
+            className="iframe"
+            title="szopol"
+            src={
+              "http://localhost:8080/games/" +
```

Comment:
Please use string literals for string concatenation eg: 
const a = 'test'; 
const b = \`my awesome text ${a}`;